src/apic-extension/azext_apic_extension/tests/latest/test_register_command.py

```diff
@@ -47,7 +47,7 @@ def test_register_with_yml_spec(self):
             self.check('description', 'API Description'), # default value when spec does not have description
             self.check('name', 'openapi'), # hard coded when spec is swagger or openapi
             self.check('specification.name', 'openapi'),
-            self.check('specification.version', '3-0-0'),
+            self.check('specification.version', '3.0.0'),
```

Comment:
What causes the behavior change?

Reply:
I would suggest keep previous behavior if no explicit ask to change it. Not sure whether users rely on existing naming convention to do something.

Reply:
It is from partner David's ask.